test/e2e/deployimage/plugin_cluster_test.go

```diff
@@ -189,7 +189,7 @@ func Run(kbc *utils.TestContext, imageCR string) {
 	ExpectWithOffset(1, err).NotTo(HaveOccurred())
 
 	By("building the controller image")
-	err = kbc.Make("docker-build", "IMG="+kbc.ImageName)
+	err = kbc.Make("docker-build", "-t", "IMG="+kbc.ImageName)
```

Comment:
@camilamacedo86 Added -t argument in docker build.

Reply:
No, we are calling the targer `make docker-build`, see: https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v3/Makefile#L72-L74

The target calls another makefile target test and the test will need the bin which will not work on prow. Why? it will try to install things and that is not allowed in the same way.

The goal here is only to build the image.
So, that to solve the issue the change is instead of we call the make target test we just execute the docker command: https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v3/Makefile#L74


We n